backend/internal/app/suggestion.go

```diff
@@ -41,7 +41,7 @@ func getSuggestions() ([]Suggestion, error) {
 		return nil, err
 	}
 
-	var suggestions []Suggestion
+	var suggestions []Suggestion = []Suggestion{}
 	for rows.Next() {
```

Comment:
Is this correct syntax??

Reply:
Yes. In the old version, if there are no suggestions, the function will return `null` since `suggestions` has not been instantiated. Now in the new version, it will return `[]` instead.

When the fronted receives `null` instead of a list, it crashes..